card/mod/standard/set/all/rich_html/toolbar.rb

```diff
@@ -34,24 +35,29 @@ def default_toolbar_args args
   # end
 
   def expanded_close_link
-    close_link "hidden-xs navbar-right"
+    opts = {}
+    opts[:no_nav] = true
+    close_link "hidden-xs pull-right navbar-text"
   end
 
   def collapsed_close_link
-    close_link "pull-right visible-xs"
+    opts = {}
+    opts[:no_nav] = true
+    close_link "pull-right visible-xs navbar-text", opts
   end
 
   def tool_navbar
     navbar "toolbar-#{card.cardname.safe_key}-#{voo.home_view}",
            toggle_align: :left, class: "slotter toolbar",
            navbar_type: "inverse",
+           no_collapse: true,
```

Comment:
General parameters like `:class` and `id` should not be ignored if you set `no_collapse` to true. It's ok that contradictory parameters like `collapsed_content` have no effect but it doesn't make sense to keep them here in the call.